db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2018_05_09_191506) do
+ActiveRecord::Schema.define(version: 2018_05_09_205449) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "recipes", force: :cascade do |t|
+    t.string "title"
+    t.text "decsription"
+    t.text "ingredients"
+    t.text "directions"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
```

Comment:
Why is this part of the PR? Please stay on top of your git flow. Not a blocker but should not be here if you always make sure to have the latest code.

Reply:
We'll be more vigilant in the future